src/main/kotlin/no/nav/omsorgspenger/soknad/KomplettSøknad.kt

```diff
@@ -15,9 +14,9 @@ data class KomplettSøknad(
     val barn: Barn,
     val relasjonTilBarnet: SøkerBarnRelasjon? = null,
     val sammeAdresse: Boolean?,
-    var legeerklæring: List<URL>,
-    var samværsavtale: List<URL>?,
+    var legeerklæringVedleggId: List<String>,
+    var samværsavtaleVedleggId: List<String>,
```

Comment:
Den skal kunne være nullable (?)

Reply:
Var sånn i gamle formatet ja, men vil heller forholde meg til en tom liste også slippe null. Eller burde den være null? :P

Reply:
i tilKomplettMelding håndteres det. samværsavtale?.map { it.vedleggId() } ?: listOf()

Reply:
Det spørs hva frontend sender. Hvis de ikke sender noe på det filtet, vil det kunne blir instansiert til tom liste. Men hvis de sender null, vil det feile. Sjekk hvordan det er i dialogen.

Reply:
Det er nullable fra frontend og i Søknad(). Derfor "?: listOf()." slik at det uansett blir en liste i KomplettSøknad()

Reply:
Aah trodde det var Søknad(). Da er det jo ok 👍

Reply:
Kjørte det ut 😈